const Todo = require('../models/todo');const errorHandler = require('../utils/errorHandler');const {update} = require("./todo");module.exports.getAll = async function (req, res) {    try {        const todo = await Todo.find({user: req.user.id});        res.status(200).json(todo)    } catch (e) {        errorHandler(res, e)    }};module.exports.getById = async function (req, res) {    try {        const todo = await Todo.findById(req.params.id);        res.status(200).json(todo)    } catch (e) {        errorHandler(res, e)    }};module.exports.remove = async function (req, res) {    try {        await Todo.remove({_id: req.params.id});        res.status(200).json({            message: 'Категория удалена.'        })    } catch (e) {        errorHandler(res, e)    }};module.exports.update = async function (req, res) {    const updated = {        name: req.body.name    };    try {        const todo = await Todo.findOneAndUpdate(            {_id: req.params.id},            {$set: updated},            {new: true}        );        res.status(200).json(todo)    } catch (e) {        errorHandler(res, e)    }};module.exports.create = async function (req, res) {    console.log(req.user);    console.log(req.body);    const todo = new Todo({        name: req.body.name,        user: req.user.id,    });    try {        await todo.save();        res.status(201).json(todo)    } catch (e) {        errorHandler(res, e)    }};